x-pack/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -53,19 +52,19 @@ interface SetupRequestParams {
     /**
      * Timestamp in ms since epoch
      */
-    start?: string;
+    start?: number;
```

Comment:
Why was this changed? It's not a biggie but does make the url a bit harder to read.

Reply:
Nothing changes in the URL @sqren, I created a new runtime to convert from string (ISO) to number (timestamp). That's why this is now a number.